vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/SockJSSocketBase.java

```diff
@@ -118,13 +117,18 @@ public void closeAfterSessionExpired() {
     close();
   }
 
+  @Override
+  public RoutingContext webContext() {
```

Comment:
Maybe for consistency rename to `webRoutingContext()`

Reply:
Sure, I'll update.